### DISCUSSION TITLE
How to control badger file size in jaeger docker image?

### DISCUSSION THREAD

**Discussion by joe-agent** [jaegertracing/jaeger#4018] [Q&A]
Someone had shared multiple options to configure badger file size in [here](https://github.com/jaegertracing/jaeger/issues/2266#issuecomment-638102537) like below:

```
opts.LevelOneSize = 32 << 20
opts.TableLoadingMode = options.FileIO 
opts.ValueLogLoadingMode = options.FileIO
opts.MaxTableSize = 4 << 20 
opts.NumLevelZeroTables = 1
opts.NumLevelZeroTablesStall = 2
opts.NumMemtables = 1
opts.NumVersionsToKeep = 0
opts.ValueLogFileSize = 32 << 20
```

Can it be done in docker compose for `jaegertracing/all-in-one:latest`? 
Any example is appreciated.

Thanks

**Comment by joe-agent** [comment]
@objectiser, @jarifibrahim, @vasanth29797, @yurishkuro  any suggestion please?

**Reply by yurishkuro** [reply]
From the linked post it looks like they did this directly in the code, because the post says these options are not exposed via Jaeger configs.
